src/Storage/Migration/Output/JsonFile.php

```diff
@@ -47,7 +48,7 @@ public function addRecord(array $data, $last)
         }
 
         // Generate the JSON string
-        $json = json_encode($data, JSON_PRETTY_PRINT);
+        $json = Json::dump($data, JSON_PRETTY_PRINT);
```

Comment:
That … is … "Coding while sober". 

Fixed and second parameter is G.O.N.E. … gone!